tests/Feature/RemoteWorkers.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Facades\URL;
+use Tests\TestCase;
+
+class RemoteWorkers extends TestCase
+{
+    protected function setUp() : void
+    {
+        parent::setUp();
+
+        URL::forceRootUrl('http://localhost');
```

Comment:
Is this necessary?

Reply:
Yes, these test cases fail for me locally if I remove this line. Perhaps another "CDash-in-a-subdir" issue?

```
1) Tests\Feature\RemoteWorkers::testRemoteWorkerAPIAccess
Expected response status code [200] but received 404.
```